src/routes/+page.svelte

```diff
@@ -1,3 +1,63 @@
-<main class="flex h-full items-center justify-center">
-	<h1 class="text-6xl text-blue-500">Startseite</h1>
+<script>
+	import { goto } from "$app/navigation";
+//Für Weiterleitung auf Start- bzw. Übungsseite
+	function goToUebung(){
+		goto('/uebung') 
+	}
+
+	function goToStart(){
+		goto('/raten')
+	}
+</script>
+
+<main>
+	<!-- Aufteilung des Bildschirms in zwei gleichgroße Hälften vertikal-->
+	<div class="flex flex-col h-screen bg-blue-200">
+		<!--Obere Hälfte mit dem KukuWi Logo-->
+		<div class="h-3/5 flex items-center justify-center">
+			<img src="/images/logo.png" alt="logo" class="max-w-full max-h-full object-contain">
+		</div> 
+	    <!-- Unter Hälfte mit den Buttons -->
+	    <div class="h-2/5 flex">
+	        <!--Unter Hälfte wird nochmal geteilt für Trainings- und Übungsbutton-->
+				<div class="w-1/2 flex items-center justify-center">
+					<div class="flex flex-col items-center rounded-2xl">
+					  <button class="flex flex-col items-center justify-center
```

Comment:
gerade bei so langen stylings, die mehrmals 1:1 so vorkommen, lohnt es sich diese als Konstante in <script> auszulagern (dient der Übersichtlichkeit) 
(Wäre an anderen Stellen des Codes auch sinnvoll)